### PR TITLE
Pan globe with touch events

### PR DESCRIPTION
**Pull request by jage**
Does not support pinch (think it's hard without something like [hammer.js](http://hammerjs.github.io/)).

Tested through Chrome emulation and with my iPhone 5s.

### CODE REVIEW

File: js/globe.js [hunk 1]
```diff
@@ -294,6 +299,38 @@ Globe = function(container, opts) {
     container.style.cursor = 'move';
   }
 
+  function onTouchDown(event) {
+    event.preventDefault();
+
+    container.addEventListener('touchmove', onTouchMove, false);
+    container.addEventListener('touchend', onTouchEnd, false);
+    container.addEventListener('touchleave', onTouchEnd, false);
+    container.addEventListener('touchcancel', onTouchEnd, false);
```
Comment (jage) [inline js/globe.js:308]: Not sure about `touchleave` and `touchcancel`, couldn't test them. But why not ...

File: js/globe.js [hunk 2]
```diff
@@ -294,6 +299,38 @@ Globe = function(container, opts) {
...
+
+    targetOnDown.x = target.x;
+    targetOnDown.y = target.y;
+  }
+
+  function onTouchMove(event) {
+    mouse.x = -event.touches[0].clientX;
+    mouse.y = event.touches[0].clientY;
+
+    var zoomDamp = distance/2000;
```
Comment (jage) [inline js/globe.js:321]: Tried with the same zoomDamp as for the mouse first, but thought it was too fast.

**Comment by roback** [comment]
Tested on chrome 46 on Android, works great :+1:

**Comment by jage** [comment]
Woho! :)

**Pull request merged**